events/carolina.md

```diff
@@ -1,9 +1,34 @@
-## Carolina's public speaking experiences
+# Carolina Ferreira
 
- - [Back to the living room hackathon](https://backtothelivingroom.tech/) (05/02/2021)
-   MeiliSearch: the future of search engines
-   https://www.youtube.com/watch?v=4aiUCYEOVgo
-	   
- - [Rawkode Live](https://www.youtube.com/channel/UCrber_mFvp_FEF7D9u8PDEA) (16/02/2021)
+![IMG-20190926-WA0000](https://user-images.githubusercontent.com/48251481/109692713-2071d880-7b89-11eb-94d5-444da042c88a.jpg)
+
+As a Developer Advocate, Carolina Ferreira helps developers have the best possible experience with the open-source search engine MeiliSearch. A former teacher turned developer, Carolina brings her patience and experience in the classroom to her new role in the tech industry. She believes that success comes hand in hand with helping, sharing, and elevating others.
+
+## Topics
+
+TBD
+
+## Experience
+
+- [Rawkode Live](https://www.youtube.com/channel/UCrber_mFvp_FEF7D9u8PDEA) (2021/02/16)
    Introduction to MeiliSearch
    https://www.youtube.com/watch?v=SJl2UWfy1nk
+   
+- [Back to the living room hackathon](https://backtothelivingroom.tech/) (20201/02/05) 
+   MeiliSearch: the future of search engines
+   https://www.youtube.com/watch?v=4aiUCYEOVgo
+
+## Links
+
+https://github.com/CaroFG
+https://www.linkedin.com/in/caro-fg/
+https://twitter.com/CarolainFG
+
+
+## Requirements
+
+- Your event is welcoming everyone, no matter where they come from, their social status, gender, orientation, beliefs, education and their mental or physical abilities;
+- Your conference provide access for wheelchair users and people with reduced mobility;
+- You have an enforced Code of Conduct (looking for one, check confcodeofconduct.com);
+- If you record my talk, the video needs to be accessible for free to everyone and cannot be sold in any capacity;
+- You want me to speak at your event because you find my content valuable for your attendees, not because you want my employer to sponsor your event.
```

Comment:
Love these!

Reply:
It's all from Fred's requirements. I just took what fitted me. But I agree, it's great :)

Reply:
FYI, my list is at https://github.com/fharper/speakerkit